go/host/db/batches.go

```diff
@@ -0,0 +1,112 @@
+package db
+
+import (
+	"bytes"
+	"math/big"
+
+	gethcommon "github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/ethdb"
+	"github.com/ethereum/go-ethereum/rlp"
+	"github.com/obscuronet/go-obscuro/go/common"
+	"github.com/obscuronet/go-obscuro/go/common/log"
+)
+
+// DB methods relating to batches.
+
+// headerKey = batchHeaderPrefix  + hash
+func batchHeaderKey(hash gethcommon.Hash) []byte {
+	return append(batchHeaderPrefix, hash.Bytes()...)
+}
+
+// GetHeadBatchHeader returns the header of the node's current head batch, or (nil, false) if no such header is found.
+func (db *DB) GetHeadBatchHeader() (*common.Header, bool) {
+	headBatchHash, found := db.readHeadBatchHash(db.kvStore)
```

Comment:
No need to have a method of type and pass a property of type.
```suggestion
	headBatchHash, found := readHeadBatchHash(db.kvStore)
```
OR
```suggestion
	headBatchHash, found := db.readHeadBatchHash()
```

Reply:
Thanks, fixing.